tasks/main.yml

```diff
@@ -16,6 +16,17 @@
     url: https://deb.nodesource.com/gpgkey/nodesource.gpg.key
     state: present
 
+- name: Check if nodejs_version is 4.x or higher
+  set_fact: debian_repo_version="4.x"
```

Comment:
I wonder if there is a way to make this more dynamic for versions 4.x and up.  But this solution works for now, just need to stay on top of this role for every new major release.